Model/Project.java

```diff
@@ -117,4 +143,10 @@ public void setIsActive(boolean isActive){
         this.isActive = isActive;
     }
 
+    public void setFavourite(boolean favourite) {
+        this.favourite = favourite;
+    }
+
+    public void setStudent(String studentId, String studentName) {
```

Comment:
bro what to do with this im not sure

Reply:
the set student we are using studentassigned id and student assigned name is it the same thing?

Reply:
Yes they are the assigned student's id and name, but we do not store the Student object itself